lib/ansible/plugins/strategy/linear.py

```diff
@@ -213,7 +213,12 @@ def run(self, iterator, play_context):
                                 skip_rest = True
                                 break
 
-                        run_once = templar.template(task.run_once) or action and getattr(action, 'BYPASS_HOST_LOOP', False)
+                        templated_task_run_once = templar.template(task.run_once)
```

Comment:
i'm not sure why you need to template, check the template value and then template again (get_validated_value already templates).

Reply:
you might want to use `templar.is_template` if what you are trying to determine is if the value itself is templatable.